rust/sealevel/libraries/hyperlane-sealevel-token/src/instruction.rs

```diff
@@ -200,3 +200,33 @@ pub fn transfer_ownership_instruction(
 
     Ok(instruction)
 }
+
+/// Transfers ownership.
+pub fn set_igp_instruction(
+    program_id: Pubkey,
+    owner_payer: Pubkey,
+    igp_program: Pubkey,
```

Comment:
Maybe worth this taking in `Option<(Pubkey, InterchainGasPaymasterType)>` so that we can also use this to unset an IGP if we ever want to in the future? But in the sealevel client, we can always set it to `Some` - just is nice to have maximal flexibility here